src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -6,13 +6,41 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private Car car;
+
+    public Driver(Long id, String name, String licenseNumber, Car car) {
+        this.id = id;
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+        this.car = car;
+    }
 
     public Driver(Long id, String name, String licenseNumber) {
         this.id = id;
         this.name = name;
         this.licenseNumber = licenseNumber;
     }
 
+    public Driver(String name, String licenseNumber, Car car) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+        this.car = car;
+    }
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
+    public Car getCar() {
```

Comment:
Remove the getCar() method, as the relationship between Driver and Car should not be in the Driver class.